.github/workflows/R-CMD-check.yaml

```diff
@@ -37,11 +38,21 @@ jobs:
           default: true
 
       - name: Configure Windows
-        if: startsWith(runner.os, 'Windows')
+        if: startsWith(runner.os, 'Windows') && matrix.config.r == 'release'
```

Comment:
May I suggest to write it as `!= 'devel'`?

Reply:
Oops, yes... Thanks for catching!